packages/mdc-floating-label/_mixins.scss

```diff
@@ -56,14 +56,14 @@
 
 @mixin mdc-floating-label-float-position($positionY, $positionX: 0%, $scale: .75) {
   .mdc-floating-label--float-above {
-    @if $positionX == 0 {
-      transform: translateY(-1 * $positionY) scale($scale);
-    } @else {
+    @if $positionX > 0 {
```

Comment:
Is there a valid use case for clients to pass a negative value here?

Reply:
There isn't presently, but I could see this becoming a thing depending on how we fix floating label in heavily-rounded outlined text fields.